src/config_cache.cpp

```diff
@@ -78,7 +78,6 @@ void config_cache::clear_defines()
 #ifdef __APPLE__
 	defines_map_["APPLE"] = preproc_define();
 
-#define MOUSE_TOUCH_EMULATION
```

Comment:
Well, if it's useful, I don't see much reason to remove it... unless its usefulness has expired or something...